src/Scanner.js

```diff
@@ -57,7 +57,19 @@ class DefaultScanner {
         this.device = device
         this.matching = createMatching()
         this.options = options
+        /** @type {number} A timestamp of the  */
+        this.initTimeStamp = Date.now()
     }
+
+    /**
+     * Determine whether we should fire the credentials autoprompt. This is needed because some sites are blank
+     * on page load and load scripts asynchronously, so our initial scan didn't set the autoprompt correctly
+     * @returns {boolean}
+     */
+    get shouldAutoprompt () {
+        return Date.now() - this.initTimeStamp <= 1500
+    }
```

Comment:
I moved from a `shouldAutoprompt` parameter that was passed around only on the first page scan, to this dynamic value. No worries, though: the actual prompt can only fire once per page, no matter how many forms it has. It's guarded later on in the code so nothing changes on that front.